frontend/src/common/enums/ModalTypes.ts

```diff
@@ -0,0 +1,3 @@
+export enum ModalTypes {
+  editProfile = 'EDIT_PROFILE',
```

Comment:
```suggestion
  EditProfile
```

no need to use string enums here